### DISCUSSION TITLE
How to increase display resolution for mimxrt1064-evk ?

### DISCUSSION THREAD

**Discussion by glory-man** [zephyrproject-rtos/zephyr#31590] [Q&A]
I have mimxrt1064 evaluation board and it work well (lvgl samples) with default (4.3 inch) display. But now I want to use greater display with resolution 800x480 with the same connection interface.
I tried sample\lvgl on board with connected display. App workes but only part of lcd is used.
What should I do to configure zephyr for such display?

**Comment by DerekSnell** [comment]
Hi @glory-man ,

Sorry, I know this an old question.  But it sounds like this is an LVGL question, instead of a Zephyr question.  Since you are using LVGL on NXP silicon, I'll make you aware NXP offers a GUI graphical development tool for LVGL called [GUI Guider](https://www.nxp.com/design/software/development-software/gui-guider:GUI-GUIDER).  Among many features, it allows you to configure the resolution of the display.  For those new to LVGL, it is a helpful tool to get started with and learn more about using LVGL.

Best regards

**Reply by glory-man** [reply]
Hi, @DerekSnell
Zephyr has an options to configure lvgl resolution, while graphic subsystem (elcdif driver) should not be Configured. I added kconfig options and fix elcdif driver in accordance of it, even programmed display orientation.
